package.json

```diff
@@ -60,6 +60,7 @@
     "lodash": "^4.17.4",
     "moment": "latest",
     "prop-types": "^15.5.10",
+    "qs": "*",
```

Comment:
This is _not_ a new dependency, it's one of react-natives. However, in me using it in the app, it's moved from being a transitive dependency to a first-order dependency. So it's noted here.